analyses/0011-keda/keda-implementation.md

```diff
@@ -118,22 +119,22 @@ Here is a proposed outline for the tech doc Table of Contents:
   - [Migrating to a new release](https://keda.sh/docs/2.13/migration/) (current
     "Migration Guide")
   - Caching Metrics
-    (https://keda.sh/docs/2.13/concepts/scaling-deployments/#caching-metrics)
+    (<https://keda.sh/docs/2.13/concepts/scaling-deployments/#caching-metrics>)
   - Pausing Autoscaling of deployments
-    (https://keda.sh/docs/2.13/concepts/scaling-deployments/#pause-autoscaling)
+    (<https://keda.sh/docs/2.13/concepts/scaling-deployments/#pause-autoscaling>)
   - Pausing Autoscaling of jobs
-    (https://keda.sh/docs/2.13/concepts/scaling-jobs/#pause-autoscaling)
-  - [Troubleshooting](https://keda.sh/docs/2.13/concepts/troubleshooting/,
-    /docs/2.13/troubleshooting/)
+    (<https://keda.sh/docs/2.13/concepts/scaling-jobs/#pause-autoscaling>)
+  - [Troubleshooting](https://keda.sh/docs/2.13/concepts/troubleshooting/),
+    `/docs/2.13/troubleshooting/`
```

Comment:
@nate-double-u FYI: the current markdown linter rules report bare URLs, and use that config because it often finds errors like these, of misformatted links.